statistics_calculations.py

```diff
@@ -70,6 +70,50 @@ def counter_by_user(df, media_df):
     counter["keilu"] = count_word(df, "כאילו").iloc[:, 1]
     counter = counter.transpose()
     counter["total"] = counter.sum(axis=1)
+    counter = counter.fillna(0)
     counter = counter.astype(int)
 
     return counter
+
+def plot_percentage(counter):
+    """
+    generates a horizontal bar plot of each category, and the percentage of each user in it.
+    counter is a df that contains the user columns + total column, and indexes of the categories
+    """
+
+    # reverse hebrew indexes
+    counter.index = counter.index.where(~counter.index.str.contains(HEBREW_LETTERS), counter.index.str[::-1])

```

Comment:
I know currently we are working with Hebrew mainly, but isn't it better practice to create a solution that will switch every right to left language? 
Isn't of great importance now, just something to think about

Reply:
you're probably right, but I think all our code will need more readjustments if we'll introduce new languages (hhh, keilu, curses). It might be better to add this when we need it.